aspnetcore/tutorials/build-your-first-blazor-app.md

```diff
@@ -22,7 +22,7 @@ Follow the guidance in the <xref:blazor/get-started> article to create a Blazor
 
 1. Browse to each of the app's three pages in the *Pages* folder: Home, Counter, and Fetch data. These pages are implemented by the Razor component files *Index.razor*, *Counter.razor*, and *FetchData.razor*.
 
-1. On the Counter page, select the **Click me** button to increment the counter without a page refresh. Incrementing a counter in a webpage normally requires writing JavaScript, but Blazor provides a better approach using C#.
+1. On the Counter page, select the **Click me** button to increment the counter without a page refresh. Incrementing a counter in a webpage normally requires writing code in two languages, JavaScript and C#. Blazor provides a better approach because only one language, C#, is used client- and server-side.
```

Comment:
I recommend avoiding the word "better". It's very much a subjective term, and we don't want this to be marketing material.

Reply:
Dan's word ........ I don't generally change Dan's words on 💀 *pain of death* 💀. :smile: lol

Surely, it says the same thing if it's just ...

> Blazor provides an approach that only requires one language, C#.

... and we'll let the reader decide if that's "better." *Stand-by!*

Reply:
**_Done!_**

Reply:
I have many opinions, but I don't guarantee that any of them are right 😃. I agree that removing "better" here makes sense.